components/hubspot/hubspot.app.mjs

```diff
@@ -185,6 +182,22 @@ export default {
         };
       },
     },
+    workflow: {
+      type: "string",
+      label: "Workflow",
+      description: "The ID of the workflow you wish to see metadata for.",
+      async options() {
```

Comment:
Should this support pagination with `context`?

Reply:
this API doesn't support pagination